resources/js/views/Teams/TeamsDashboard.vue

```diff
@@ -0,0 +1,117 @@
+<template>
+    <section class="tdc">
+        <p class="subtitle is-centered is-3">Teams Dashboard</p>
+        <p class="mb1">Here you can find the combined impact made by all the teams you have joined.</p>
+
+        <div class="columns">
+            <div class="column teams-card">
+                <span class="title is-2" style="color: #7b848e;">{{ photos_count }}</span>
```

Comment:
Should define constant for color.

Reply:
We will! Experimenting at the moment was trying to move fast